### DISCUSSION TITLE
Taxonomy Term Visit URL link?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#5993] [General]
In a taxonomy term, is it possible to customise the `Visit URL` button url? 

By default it links to `taxonomy-name/term-name` but I don't have any templates at that location. I'm only using this taxonomy with a collection called `products`. The URL to the taxonomy term is  `/products/taxonomy-name/term-name`

**Comment by duncanmcclean** [comment]
No, I don't think you can change it. 

What I usually do is create a redirect for its URL to my actual URL for the taxonomy term, basically like this:

```php
// routes/web.php

Route::get('festival/{slug}', function ($slug) {
    return redirect()->to('whats-on/festival/' . $slug);
});
```
